packages/material-ui/src/Modal/manageAriaHidden.js

```diff
@@ -4,10 +4,10 @@ function isHidable(node) {
   return node.nodeType === 1 && BLACKLIST.indexOf(node.tagName.toLowerCase()) === -1;
 }
 
-function siblings(container, mount, callback) {
+function siblings(container, mount, callback, currentNode) {
   mount = [].concat(mount); // eslint-disable-line no-param-reassign
   [].forEach.call(container.children, node => {
-    if (mount.indexOf(node) === -1 && isHidable(node)) {
+    if (mount.indexOf(node) === -1 && node !== currentNode && isHidable(node)) {
```

Comment:
I think `mount.indexOf` wanted to do the same as `node !== currentNode`. Could you find out what the difference between those two expressions is and add some documentation? That might be the actual bug here.

Reply:
That mount.indexOf works if we give it the currentNode instead of mountNode. Is that desired?